test/verify/check-realms

```diff
@@ -59,6 +63,15 @@ for x in $(seq 1 60); do
 done
 """
 
+# https://en.wikipedia.org/wiki/HMAC-based_One-time_Password_algorithm
+# https://stackoverflow.com/questions/8529265/google-authenticator-implementation-in-python
+def totp_token(secret, counter, digits=6, hash_alg=hashlib.sha1):
```

Comment:
Ugh, everything here is *H*OTP, same for the comments and variables below. I'll push a fix after the tests finished, and maybe @Gundersanne also has other comments.

Reply:
Tests seem fine, I pushed the fixed name.